scripts/install-helm.sh

```diff
@@ -0,0 +1,128 @@
+#!/bin/bash -e
+
+#Based on ideas from https://github.com/rook/rook/blob/master/tests/scripts/helm.sh
+
+temp="/tmp/ceph-csi-tests-scripts-helm"
```

Comment:
please use a real tmp dir if you can (not sure how to do the cleanup in that case)
```
temp=$(mktemp -d)
```

Also try not to mix lowercase and UPPERCASE variables.